cluster/charts/crossplane/templates/stack-manager-deployment.yaml

```diff
@@ -21,6 +21,9 @@ spec:
         app: {{ template "name" . }}-stack-manager
         release: {{ .Release.Name }}
     spec:
+      {{- if .Values.priorityClassName }}
+      priorityClassName: {{ .Values.priorityClassName }}
```

Comment:
Nitpick: I don't really know why we do it, but it looks like other strings we conditionally add get piped to `quote`. Do we want to do that here also? Or maybe we should remove it in other places?

Reply:
We are doing that if that field could get a value which could be parsed as `int` by yaml wheres actual type should be a `string`.  It might not really be needed where you don't expect an int as parameter but would not hurt to have it. So, I'll add it.

Reply:
Awesome, thanks!